public/src/components/account/account.jsx

```diff
@@ -41,7 +41,7 @@ export class AccountPage extends Component {
         <h6>Phone:</h6>
         <h3>{account_info ? phone_number : ''}</h3>
         <div className="text-xs-right">
-          <Link className="btn btn-primary" to={`/account/${id}/edit`}>
+          <Link className="btn btn-primary" to={is_admin ? `/admin/account/${id}/edit` : `/volunteers/account/${id}/edit`} replace>
```

Comment:
I'm curious. What's the `replace` at the end of this link for?

Reply:
Replace is meant to be a bool. When true, clicking the link will replace the current entry in the history stack instead of adding a new one. It was something I was trying out before I went with the is_admin check. I removed it.